src/wilcoxon.jl

```diff
@@ -163,6 +163,27 @@ function StatsAPI.pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+function psignrank(W, n, lower_tail)
+    int_type = typeof(n)
+    W = int_type(W)
+    two = one(int_type) + one(int_type)
+    max_W =  n * (n + one(int_type)) ÷ two + one(int_type)
```

Comment:
Is it actually possible to construct a `Vector` with non-`Int` length? I assume this could just always be an `Int` as it doesn't have any ilpact on the calculated values.